Modules/BenchPress.Azure/Public/Confirm-StorageAccount.ps1

```diff
@@ -0,0 +1,49 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-StorageAccount {
+  <#
+    .SYNOPSIS
+      Confirms that a Storage Account exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPStorageAccount cmdlet gets a Storage Account using the specified Storage Account and
+      Resource Group name.
+
+    .PARAMETER Name
+      The name of the Storage Account
+
+    .PARAMETER ResourceGroupName
+      The name of the Resource Group
+
+    .EXAMPLE
+      Confirm-AzBPStorageAccount -Name "teststorageaccount" -ResourceGroupName "rgbenchpresstest"
+
+    .INPUTS
+      System.String
+
+    .OUTPUTS
+      ConfirmResult
+  #>
+  [CmdletBinding()]
+  [OutputType([ConfirmResult])]
+  param (
+    [Parameter(Mandatory=$true)]
+    [string]$ServerName,
```

Comment:
This parameter should be renamed to `Name`.

Reply:
good catch, thanks 👍